services/showcase/api/endpoints/routes/line_of_business_issuer.py

```diff
@@ -50,7 +50,8 @@ async def issue_degree(
     # we will need some info for revoking this (rev_reg_id, cred_rev_id)
     logger.info("issue credential response")
     logger.info(resp)
-    return resp
```

Comment:
The issue was the internal showcase-api -> traction-api call was throwing a 404, and the showcase-api was returning that 404 as the body of a 200 response.

The the showcase FE successfully receives a response.. then the body of the response is actually an error message.

Reply:
but this was working... so what really changed? does that mean the traction v0 changed? it's now throwing a different response in the same situation? if the fix needs to be in showcase, it needs to be done and we re-deploy it all but seems like there is new behaviour in v0 - when there shouldn't be any change.

Reply:
the comma is a `flask` short hand to set the status code of the response, but doesn't look like fastapi like that pattern anyways, reverted

Reply:
👍